CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ endif()
 # Options
 option(TRITON_BUILD_TUTORIALS "Build C++ Triton tutorials" ON)
 option(TRITON_BUILD_PYTHON_MODULE "Build Python Triton bindings" OFF)
+set(TRITON_CODEGEN_BACKENDS "" CACHE STRING "Enable different codegen backends")
```

Comment:
`TRITON_CODEGEN_BACKENDS` is a list to represent different backends. It could be something like "intel_xpu_backend;amd_hip_backend".